data/en/cachekeyexists.json

```diff
@@ -4,14 +4,14 @@
 	"syntax": "cacheKeyExists(key [, cacheName])",
 	"returns": "boolean",
 	"related": [],
-	"description": "",
+	"description": "This function is deprecated, use cacheIdExists instead. Returns true/false whether the cache contains an element with the specified keyname.",
 	"params": [
-		{"name": "key", "description": "", "required": true, "default": "", "type": "string", "values": []},
-		{"name": "cacheName", "description": "", "required": false, "default": "", "type": "string", "values": []}
+		{"name": "key", "description": "Key to check", "required": true, "default": "", "type": "string", "values": []},
+		{"name": "cacheName", "description": "Definition of the cache used by name, when not set the \"default Object Cache\" defined in Lucee Administrator is used instead.", "required": false, "default": "", "type": "string", "values": []}
 	],
 	"engines": {
 		"lucee": {"minimum_version": "", "notes": "", "docs": "http://docs.lucee.org/reference/functions/cachekeyexists.html"}
```

Comment:
We have a special way to indicate that something is deprecated, you should add `"deprecated":"4.5"` to the `"lucee"` engine object. Assuming it was deprecated in Lucee 4.5

Reply:
Sorry about that. I tried searching the Lucee documentation for when it was deprecated but cannot find any of the previous version docs. I would assume it was 4.5 as well. Should I just add that in or take it out completely?

Reply:
Yeah let's just add in 4.5, if someone finds otherwise it can be updated later. Just by doing that it will automatically add a alert box to the docs showing it is deprecated, for example: https://cfdocs.org/parameterexists